### ISSUE TITLE
[Bug Bash][Test Failure][Accessibility]No dashed-line box show for the check-box “Top-Level” in “Migrate NuGet format to PackageReference” dialog when navigating by Tab

### ISSUE THREAD

**Issue by v-luzh** [NuGet/Home#8685]
## Details about Problem
NuGet version: NuGet Client Dev\5.4.0.6265
VS Version: Master\29410.161
OS: Windows-10-Enterprise-19H2(18363.418)
## Detailed repro steps 
1. Create a C# Console Application (.NET Framework) template via VS.
2. Open PM UI of the project by right-clicking the project->Manage NuGet Packages… in Solution Explorer.
3. Install a package with “packages.config” format.
4. Right-click the “References” node and select the “Migrate packages.config to PackageReference…”
5. Tab through the “Migrate NuGet format to PackageReference” dialog and verify the check-box “Top-Level”.
## Expected
There should be a dashed-line box show for the check-box “Top-Level” when tabbing to it. Please refer to the same behavior for the check-box “Include prerelease” when navigating by Tab in PM UI as below screenshot.
![CorrectSelection](https://user-images.githubusercontent.com/20632625/66645471-1ed30800-ec56-11e9-864a-9eeb2fbb4428.png)
## Actual
No dashed-line box show for the check-box “Top-Level” when tabbing to it as below screenshot. 
![NoDashlinedBox](https://user-images.githubusercontent.com/20632625/66645490-2abeca00-ec56-11e9-9cb6-618941cc1dd9.png)
